app/views/staff/activities/details.html.haml

```diff
@@ -27,22 +27,20 @@
             = link_to t("tabs.activity.details"),
               organisation_activity_details_path(@activity.organisation, @activity),
               { class: "govuk-tabs__tab", role: "tab", tabindex: -1, aria: { controls: "details", selected: false } }
+          - if !@activity.third_party_project?
```

Comment:
Hmmm, so at any level, if there are no children there is no 'Child activities' tab? 

It's like the  discussion we had before, I can't remember what it was about, but it boils down to somehing like: is it better to show the empty thing so the user can see 'why' it's empty (no children) vs not seeing it at all and perhaps wondering 'why is there no child activities tab that I see on other activities' – I feel like I prefer the show the empty thing (by a tiny amount) but as you say, we should have an empty state to explain.

I am happy either way, I have a preference but wouldn't argue over it - I'll change it to use `children` and we can always change it back if we feel like it's a bad move (and add the empty state for the table at the same time).

Reply:
So there is a sort of reason to show the children tab at the moment, this is where a lot of the specs expect the 'create activity' button to appear. It will be a big chunk of change to refactor them all once the tab disappears.

I would like to keep this PR from growing further so I propose I create a new card to remove all the 'create activity' buttons, forcing the user to always use the one at /activities and then switch the children tabs to be shown only when there are children – how does that sit with you @tahb?

Reply:
It's close but I share your view. Better to always have the tab and for it to be empty for consistency sake. It'll be easier to add in a 'no content' scenario to that page and keeping the tab around will help us in moving the create activity buttons - a good reason.

I agree. I was thinking we'd end up with a single create button on the activities page and a card in the backlog to capture that would be helpful 👍 As the buttons remain functional for now I'd be in favour of leaving it for another sprint so that we can focus more time on the submission jourey.